docs/_includes/_nav.njk

```diff
@@ -12,48 +12,25 @@
             <img alt="PatternFly Elements"
                  src="{{ '/images/pfe-logo-icon-white.svg' | url }}"
                  width="57.5" height="57.5">
-          </picture>  
+          </picture>
         </a>
       </div>
-      {# TODO: implement pf-dropdown
-      <pf-dropdown id="docs-versions-dropdown" aria-label="Docs Versions" label="Docs Versions">
+      <pf-dropdown id="docs-versions-dropdown">
+        <pf-button slot="trigger"
+                   variant="control"
+                   icon="caret-down"
+                   icon-set="fas">Versions</pf-button>
```

Comment:
Given if the prior comment is correct I think the button is already handled by `pf-dropdown` instead it should have the ability to slot in text into that button

```Suggestion
        <span slot="button-text">Versions</span>
```

Reply:
Currently this is outputting Dropdown:

![image](https://github.com/patternfly/patternfly-elements/assets/13666/947d3dfc-ec5a-4e63-a5d8-2affdfc73809)

Reply:
Helps if I read notes to reviewers huh?  🙈